apps/super-app/src/common/components/navbar/NavIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import { NavLink } from "react-router-dom";
+function NavIcon(props:any) {
+  return (
+    <div>
+      <NavLink
+        to={props.link}
+        className={({ isActive }) =>
+          isActive ? "navbar-icon-active" : "navbar-icon"
+        }
+      >
+        {props.icon}
+        <span className="navbar-icon-text">{props.text}</span>
+      </NavLink>
+    </div>
+  );
+}
+export default NavIcon;
```

Comment:
I suggest putting this in the same file as Navbar. We're never going to use this outside of Navbar anyway :)